README.md

```diff
@@ -466,5 +467,5 @@ repositories {
 
 This project is not affiliated with or endorsed by the Jetbrains.  
 All code, unless specified otherwise, is licensed under the [MIT](https://opensource.org/licenses/MIT) license.  
-Copyright (c) 2019 Pinterest, Inc.
+Copyright (c) 2019 Pinterest, Inc.  
```

Comment:
what's the space at the end of the line here for?

Reply:
@NightlyNexus otherwise there is no linebreak between the copyright notices. Markdown is weird.

Reply:
`</br>` works in markdown, for what that's worth.